src/components/data-exploration/CellInfo.jsx

```diff
@@ -37,9 +20,9 @@ const CellInfo = (props) => {
         pointerEvents: 'none',
       }}
     >
-      {cellInfo.cellName ? (
+      {cellInfo.cellId ? (
         <div style={cellInfoStyle}>
-          {`Cell id: ${cellInfo.cellName}`}
+          {`Cell id: ${cellInfo.cellId}`}
         </div>
       ) : <></>}
       {cellInfo.geneName ? (
```

Comment:
Sorry, just had another look. Can you see if we need these `cellInfo.cellId`, `cellInfo.expression`, `cellInfo....` checks? Are there cases where cellInfo exists but some of these properties do not / partially exist? Otherwise I don't think we need these checks,

Reply:
Yeah sometimes we don't have them that's why i kept it